addons/html_editor/static/src/main/font/color_plugin.js

```diff
@@ -337,11 +335,12 @@ export class ColorPlugin extends Plugin {
      * @param {'color'|'backgroundColor'} mode 'color' or 'backgroundColor'
      */
     colorElement(element, color, mode) {
-        const newClassName = element.className
+        const oldClassName = element.getAttribute("class") || "";
```

Comment:
If the element is a `path` for example (not standard html element but svg element)